Fix configuration for development and test

pabloalba opened the pull request:
With the current code, on the next step the user won't be able to execute the tests on H2, because the tutorial suggest to change the global database driver.
I propose to change only the 'development' configuration to mysql. This allows the user to use mysql on development and H2 en test.

pabloalba committed:
Fix configuration for development and test

With the current code, on the next step the user won't be able to execute the tests on H2, because the tutorial suggest to change the global database driver.
I propose to change only the 'development' configuration to mysql. This allows the user to use mysql on development and H2 en test.

graemerocher commented on src/main/docs/guide/configure.adoc:
```diff
@@ -19,18 +19,7 @@ And then altering the configuration found in `grails-app/conf/application.yml`.
 include::{sourceDir}/grails-app/conf/application.yml[tags=dataSourceConfig,indent=0]
 ----
 
-To alter this for MySQL you should specify the driver class name, username and password:
```
This change removes the line about altering for MySQL, need to combine the sentence above with the new content

pabloalba replied:
Fixed!

graemerocher replied:
Thanks!

pabloalba committed:
Combine the sentences

Combine the sentences to specify that the user should specify the driver class name, username and password
